### ISSUE TITLE
Weird flicker after page selection

### ISSUE THREAD

**Issue by jkwiecien** [chenupt/SpringIndicator#14]
I've noticed that selected page indicator flickers a short while after the page is selected. Sometimes even twice. Looks rather bad.

**Comment by Gerlot** [comment]
Yes, I noticed it too, it's pretty annoying after you notice it first. I think it really should be resolved, because otherwise this lib looks really good!

I also noticed that it seems to be flickering twice when you scroll through multiple pages, and only flickrs once, if you scroll through one page. And the flickering happens also when a soft input (onscreen keyboard) comes up and resizes the view.

**Comment by Gerlot** [comment]
Hmm... the weird thing that it doesn't happen in the sample app. I'm currently trying to determine the cause, but I cannot make it to work in my application like in the sampe. Besides the flickering, the page changing is also faster in my app, and the spring effect is so fast it's barely noticeable when changing a page...

**Comment by Gerlot** [comment]
Ok, I figured out. If I also use your ScrollerViewPager which is included in the library, AND call it's fixScrollSpeed() method, AND also use your multiple model library (https://github.com/chenupt/MultipleModel) the flickering doesn't happen, and the spring animation is smooth and slower.

But these requirements are not stated in the README, and why are they required anyway? What's the reason that the indicator lib cannot work with a simple Android ViewPager and FragmentPagerAdapter? Also, using your other views and lib requires using support Fragments, and then, you cannot use your own custom adapters.

Please, at least update the README, to let others now, but I would really appreciate some explanation. The effect is otherwise pretty cool, and I really like this lib.

**Comment by olliaustor** [comment]
Easy fix (will provide this as pull request soon):
Open file lib/src/main/java/github/chenupt/springindicator/SpringIndicator.java
Change method onLayout to:
```java
@Override
protected void onLayout(boolean changed, int l, int t, int r, int b) {
    super.onLayout(changed, l, t, r, b);
    if (changed || _currentView == null) createPoints();
    setSelectedTextColor(viewPager.getCurrentItem());
}
```
